.github/workflows/release.yml

```
@@ -0,0 +1,262 @@
+                  </div>
+                  
+                  <div class="footer">
+                      <p>Built with ❤️ using Storybook • Updated automatically from monorepo-release branch</p>
+                  </div>
+              </div>
+          </body>
+          </html>
+          EOF
+
+      - name: Setup Pages
+        uses: actions/configure-pages@v3
```

actionlint (1.7.7)

the runner of "actions/configure-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)